src/api/status/public/index.html

```diff
@@ -56,15 +56,15 @@
           href=" https://demos.creative-tim.com/material-dashboard/pages/dashboard "
```

Comment:
I just noticed that this `<a>` should be remove as it links to the demo page.

Reply:
I put the link directs to the telescope dashboard page. I think it can also prevent 404 page